chia/_tests/core/full_node/stores/test_coin_store.py

```diff
@@ -178,39 +178,31 @@ async def test_set_spent(db_version: int, bt: BlockTools) -> None:
 
         # Save/get block
         for block in blocks:
-            if block.is_transaction_block():
-                removals: list[bytes32] = []
-                additions: list[Coin] = []
-                async with db_wrapper.writer():
-                    if block.is_transaction_block():
-                        assert block.foliage_transaction_block is not None
-                        await coin_store.new_block(
-                            block.height,
-                            block.foliage_transaction_block.timestamp,
-                            block.get_included_reward_coins(),
-                            additions,
-                            removals,
-                        )
-
-                    coins = block.get_included_reward_coins()
-                    records = [await coin_store.get_coin_record(coin.name()) for coin in coins]
+            if not block.is_transaction_block():
+                continue
```

Comment:
this skips most of the tests in this for-loop. I think this makes the test have less coverage

Reply:
Hm? test coverage should be exactly the same before and after.